src/app/pages/create-new-test/question-list/question/question.component.ts

```diff
@@ -100,7 +134,7 @@ export class QuestionComponent implements ControlValueAccessor, OnInit {
 
   writeValue(value: Question) {
     if (value !== void(0)) {
-      this.form.setValue(value);
+      this.form.patchValue(value);
```

Comment:
Why did you rename answers to options? On back-end we have answers and if you pass value(from back-end) to this form form, you will recieve error or unexpected behavior, because form except answers field